### PR TITLE
Travisci test

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

**Comment by frederic34** [comment]
@eldy the goal of this PR is to reduce analysis of only modified php files

File: .travis.yml [hunk 1]
```diff
@@ -264,15 +264,21 @@ script:
   echo
 
 - |
+  git remote set-branches --add origin develop
+  git fetch
+  STAGED_FILES_CMD=`git diff --name-only --diff-filter=ACMR origin/develop | grep \\\\.php`
```
Comment (eldy) [inline .travis.yml:269]: Do we really need this STAGED_FILES_CMD ? It seems always empty.

File: .travis.yml [hunk 2]
```diff
@@ -264,15 +264,21 @@ script:
...
+  FILES=${FILES:-$STAGED_FILES_CMD}
```
Comment (eldy) [inline .travis.yml:270]: How does the $FILES var is filled ? What happen on large commit with 1000 files ?

Comment (frederic34) [reply .travis.yml]: @eldy I am going to make a test with a lot of files

**Comment by frederic34** [comment]
image

**Comment by eldy** [comment]
It seems it hangs travis.

**Pull request closed without merging**